python_modules/dagster/dagster/_core/definitions/declarative_automation/operators/dep_operators.py

```diff
@@ -53,6 +53,23 @@ def evaluate(self, context: AutomationContext[T_EntityKey]) -> AutomationResult[
         )
 
 
+@whitelist_for_serdes
+class AssetMatchesCondition(BuiltinAutomationCondition[T_EntityKey]):
+    to_key: T_EntityKey
```

Comment:
let's rename "to_key" to "key".

In the context of this class, if it's called "EntityMatchesCondition", then it's natural to ask "what entity key are we talking about?", which feels more natural to find under `condition.key` than `condition.to_key`. in essence, the "to" prefix makes sense in the context of the actual mapping operation, but isn't necessary here.